docs/user/dashboard/lens-percentage.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[analyze-data-as-percentage-in-line]]
+== Advanced guide: Analyze data as percentage in Lens
+
+It is often useful to with with percentages within your data: from computing a single metric about performance goal, to representing the proportion of metrics over time.
```

Comment:
I like that the benefit of percentages is here, but at this point, are users familiar with percentages? Can we include information about what percentages are?